src/test/java/suite/E2EPrivateOperatorTestSuite.java

```diff
@@ -7,7 +7,8 @@
 @Suite
 @SelectClasses({
         BasicTestSuite.class,
-        V2ApiOperatorTest.class
+        V2ApiOperatorTest.class,
+        OptoutTestSuite.class
```

Comment:
Private operators cannot log out but should respect log out. How can we test the latter only?

Reply:
This is already covered in V2ApiOperatorTest